### PR TITLE
Add rails 6 to test matrix

### LINKED ISSUE #598: Rails 6
Hi there, has anyone tested it with any of the latest beta versions?
Is it compatible, if yes what the issues are there?

The release of Rails 6.0 is planned to April 30; the compatibility questions are going to be very critical really soon.

### PR DESCRIPTION
**Pull request by tachyons**
This PR is to Add Rails 6 in test matrix and make it 💚 . Rails 6 specific enhancements are not part of this PR

Fixes #598

### CODE REVIEW

**Comment by jean-francois-labbe** [comment]
Hey, I tried this PR on my project but it deletes the development database at the end of tests.
It happens when rails 6 new feature is used: `parallelize(workers: :number_of_processors)` with apartment gem.

I tried to reproduce on a brand new rails project without apartment gem and I couldn't.

**Comment by tachyons** [comment]
Interesting. I could reproduce the issue here https://github.com/tachyons/rails6_apartment. Yet to investigate the cause

**Comment by bdtomlin** [comment]
FYI,

I wanted to help figure this out to help move the rails 6 updates further. Unfortunately I was not able to reproduce this issue with your repo at https://github.com/tachyons/rails6_apartment or with my own codebase.

**Comment by tachyons** [comment]
Just run rails test on above repo, it will fail

**Comment by bdtomlin** [comment]
Sorry, you are correct. I misread. I thought the issue was with the development db was being dropped. I'll experiment some more.

**Comment by bdtomlin** [comment]
Here's an example of how I was able to get parallel tests working with apartment on a fresh Rails 6 app: https://github.com/bdtomlin/my_apartment.

I added the following to my test_helper.rb...

```ruby
class ActiveSupport::TestCase
  parallelize_setup do
    Apartment::Tenant.drop('www') rescue nil
    Apartment::Tenant.create('www')
    Apartment::Tenant.switch! 'www'
  end

  parallelize_teardown do
    Apartment::Tenant.drop('www') rescue nil
    Apartment::Tenant.reset
  end

  # Run tests in parallel with specified workers
  parallelize(workers: :number_of_processors)

  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
  fixtures :all

  # Add more helper methods to be used by all tests here...
end
````

I know there are lot's of different ways that someone may have their app set up, but this gives the basic principles of how to make it work.

The basic premise is based on this wiki article: https://github.com/influitive/apartment/wiki/Testing-Your-Application

If something like this is a sufficient solution, maybe we can add it to the README or the wiki.

File: lib/apartment/adapters/postgresql_adapter.rb [hunk 1]
```diff
@@ -110,6 +110,8 @@ class PostgresqlSchemaFromSqlAdapter < PostgresqlSchemaAdapter
         /SET lock_timeout/i,                          # new in postgresql 9.3
         /SET row_security/i,                          # new in postgresql 9.5
         /SET idle_in_transaction_session_timeout/i,   # new in postgresql 9.6
+        /CREATE SCHEMA public/i,
+        /COMMENT ON SCHEMA public/i,
```
Comment (mikecmpbll) [inline lib/apartment/adapters/postgresql_adapter.rb]: can you let me know the rationale for these changes? i'm less knowledgeable about the pg side of things.

Comment (mikecmpbll) [reply lib/apartment/adapters/postgresql_adapter.rb]: eh; is it related to https://github.com/influitive/apartment/issues/571#issuecomment-454553287 ?

Comment (tachyons) [reply lib/apartment/adapters/postgresql_adapter.rb]: Yes

Comment (mikecmpbll) [reply lib/apartment/adapters/postgresql_adapter.rb]: i've just merged these changes from an earlier PR, can you resolve the conflicts please? :')

Comment (tachyons) [reply lib/apartment/adapters/postgresql_adapter.rb]: @mikecmpbll Done

**Pull request merged**